includes/classes/debug.php

```diff
@@ -282,7 +280,7 @@ public function do_debug_options_actions( $settings ) {
 			return $orig_settings;
 		}
 
-		wp_die( '<xmp>' . __LINE__ . ') $settings: ' . print_r( $settings, true ) . '</xmp>' . $back_button, __( 'Debug Mode', 'content-workflow-by-bynder' ) );
+		wp_die( '<xmp>' . __LINE__ . ') $settings: ' . esc_html(print_r( $settings, true )) . '</xmp>' . wp_kses_post($back_button), esc_html__( 'Debug Mode', 'content-workflow-by-bynder' ) );
```

Comment:
@alexoakland Will this escape all the `<pre>` tags?

Reply:
It will! Good spot! I'll wrap it in them to keep things simple!